lib/probes/mongodb-core.js

```diff
@@ -267,14 +269,8 @@ const dataMakers = [
     }
   }],
   [notUndef('aggregate'), function (data, cmd) {
-    const p = cmd.pipeline
-    if (p[0].$match && p[1].$group) {
-      const p1 = p[1].$group
-      if (p1._id === null && p1.n && p1.n.$sum === 1) {
-        data.QueryOp = 'count'
-        data.Query = JSON.stringify(p[0].$match)
-      }
-    }
+    data.QueryOp = 'aggregate'
+    data.Pipeline = JSON.stringify(cmd.pipeline)
```

Comment:
nice! I double-checked with the spec ;)

Reply:
I updated the spec to conform to this, so it's a little bit self-referential.